app/Libraries/ValidationErrors.php

```diff
@@ -43,6 +45,12 @@ public function add($column, $rawMessage, $params = null) : self
 
         $params['attribute'] = $column;
 
+        $attributeKey = $this->keyBase.$this->prefix.'.attributes.'.$column;
+
+        if (Lang::has($attributeKey)) {
```

Comment:
Use `trans_exists` as Lang::has doesn't handle crowdin's empty string translation.

(and maybe optionally make it so `$params['attribute']` only set once)

Reply:
Or maybe it's fine.